CavalcanteLucas/todolist
# OL-37: Boilerplate setup

**Link to task:** 
https://labcodes.atlassian.net/browse/OL-37

**How to test:**
- Verify updates on README.md
- Check if boilerplate is running locally with `make run`
- Check if boilerplate is running on the domain `todo-labc.herokuapp.com`

**Description of your solution:**
Noticed textual information that has been written for a generic boilerplate. Included section on `Deploying to Heroku` and personal info about the author on `README.md`. Also modified the frontend components with initial information about the ToDo-LabC App.
